.expeditor/config.yml

```diff
@@ -0,0 +1,38 @@
+# Documentation available at https://expeditor.chef.io/docs/getting-started/
+---
+
+pipelines:
+ - verify:
+    description: Pull Request validation tests
+    public: true
+
+slack:
+ notify_channel: inspec-notify
+
+github:
+ delete_branch_on_merge: true
+ minor_bump_labels:
+  - "Version: Bump Minor"
+ version_tag_format: v{{version}}
+ release_branch:
+  - master
+ 
+changelog:
+ categories:
+  - "Type: New Resource": "New Resources"
+  - "Type: New Feature": "New Features"
```

Comment:
For better or worse, labels in this repo use slashes, not colons- `Type/New Feature` so these will not match.